roles/multipathd/tasks/main.yml

```diff
@@ -12,6 +12,18 @@
 # License for the specific language governing permissions and limitations
 # under the License.
 
+- name: Add backports PPA
+  ansible.builtin.apt_repository:
+    repo: ppa:vexxhost/backports
```

Comment:
Minor warning from apt:

```
W: http://ppa.launchpad.net/vexxhost/backports/ubuntu/dists/jammy/InRelease: Key is stored in legacy trusted.gpg keyring (/etc/apt/trusted.gpg), see the DEPRECATION section in apt-key(8) for details.
```

I think the workaround is to grab the key directly.. something along these lines:

```
- name: Add backports PPA key
  ansible.builtin.get_url:
    url: http://ppa.launchpad.net/vexxhost/backports/ubuntu/dists/jammy/Release.gpg
    dest: /etc/apt/keyrings/ppa_vexxhost_backports.asc
```